tox.ini

```diff
@@ -5,6 +5,7 @@ skip_missing_interpreters = true
 [testenv]
 deps=
   pytest
+  mock
```

Comment:
I would prefer to avoid this and use monkeypatch https://docs.pytest.org/en/latest/monkeypatch.html

Reply:
Done